tcl/org.testeditor.tcl.dsl.ui/src/org/testeditor/tcl/dsl/ui/editor/TclModelDragAndDropUpdater.xtend

```diff
@@ -182,7 +185,8 @@ class TclModelDragAndDropUpdater {
 	 * the modification of the import section must take place before actually adding the new
 	 * test step component.
 	 */
-	public def void updateImports(TclModel tclModel, Component droppedObject, String qualifiedName) {
+	public def void updateImports(TclModel tclModel, EObject dropTarget, Component droppedObject,
```

Comment:
why is the dropTarget needed? dropping A on the first or the second context should not make a difference in respect to the import statements, right?
component A:
- do this
- do that

component full.qual.B
- do some other